modular_skyrat/modules/assault_operatives/code/assault_operatives_outfits.dm

```diff
@@ -0,0 +1,141 @@
+//KITS
+/datum/outfit/assaultops
+	name = "I couldn't choose one!"
+
+	head = /obj/item/clothing/head/helmet/swat
+	mask = /obj/item/clothing/mask/gas/syndicate
+	glasses = /obj/item/clothing/glasses/thermal
+	uniform = /obj/item/clothing/under/syndicate
+	shoes = /obj/item/clothing/shoes/combat
+	gloves =  /obj/item/clothing/gloves/combat
+	back = /obj/item/storage/backpack/fireproof
+	ears = /obj/item/radio/headset/syndicate/alt
+	l_pocket = /obj/item/modular_computer/tablet/nukeops
+	id = /obj/item/card/id/advanced/chameleon
+	back = /obj/item/mod/control/pre_equipped/nuclear
+	suit_store = /obj/item/gun/ballistic/automatic/pistol/aps
+	r_pocket = /obj/item/ammo_box/magazine/m9mm_aps
+	belt = /obj/item/storage/belt/utility/syndicate
+
+	var/command_radio = FALSE
+	var/cqc = FALSE
+
+/datum/outfit/assaultops/post_equip(mob/living/carbon/human/H)
+	var/obj/item/radio/R = H.ears
+	R.set_frequency(FREQ_SYNDICATE)
+	R.freqlock = TRUE
+	if(command_radio)
+		R.command = TRUE
+
+	if(cqc)
+		var/datum/martial_art/cqc/MA = new
+		MA.teach(H)
+
+	var/obj/item/implant/weapons_auth/W = new/obj/item/implant/weapons_auth(H)
+	W.implant(H)
+
+	H.update_icons()
+
+/datum/outfit/assaultops/cqb
+	name = "Assault Operative - CQB"
+
+	backpack_contents = list(/obj/item/storage/box/survival/syndie=1,\
+		/obj/item/knife/combat,\
+		/obj/item/gun/energy/disabler,\
+		/obj/item/gun/ballistic/automatic/c20r,\
+		/obj/item/ammo_box/magazine/smgm45=4,\
+		)
+
+	cqc = TRUE
+
+/datum/outfit/assaultops/demoman
+	name = "Assault Operative - Demolitions"
+
+	belt = /obj/item/storage/belt/grenade/full
+	backpack_contents = list(/obj/item/storage/box/survival/syndie=1,\
+		/obj/item/knife/combat,\
+		/obj/item/gun/energy/disabler,\
+		/obj/item/gun/ballistic/automatic/gyropistol,\
```

Comment:
most of these load-outs are actually, compared to old assault-ops, well balanced (besides the guns, but thats here-not-there once the station really tries to kill them) BESIDES for this loadout, the gyrojet is way-way to strong, shit like 2 taps people through the best armor against it, i'd swap it for the /obj/item/gun/grenadelauncher (along with the ammo /obj/item/ammo_box/a40mm)

Reply:
done